README.md

```diff
@@ -1,47 +1,26 @@
-[![Build Status](https://travis-ci.org/sul-dlss/ld4p-marcxml-to-bibframe1
-.svg?branch=master)](https://travis-ci.org/sul-dlss/ld4p-marcxml-to-bibframe1
-)
-[![Coverage Status](https://coveralls.io/repos/github/sul-dlss/ld4p-marcxml-to-bibframe1
-/badge.svg?branch=master)](https://coveralls.io/github/sul-dlss/ld4p-marcxml-to-bibframe1?branch=master)
-[![Dependency Status](https://gemnasium.com/badges/github.com/sul-dlss/ld4p-marcxml-to-bibframe1.svg)](https://gemnasium.com/github.com/sul-dlss/ld4p-marcxml-to-bibframe1)
```

Comment:
keep the badge for dependency -- I know it's only for capistrano ... and I decided to include it in case there was some wacko vulnerability.